tests/lib/rules/strict-component-boundaries.js

```diff
@@ -57,5 +62,10 @@ ruleTester.run('strict-component-boundaries', rule, {
       parserOptions,
       errors,
     },
+    {
+      code: `import someThing from '../SomeOtherComponent/fixtures/SomeMockQuery/query.json';`,
+      parserOptions,
+      errors,
+    },
```

Comment:
I feel like you are not testing the main thing you indicated this PR was to address